core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -1076,14 +1196,20 @@ object AutoTuner extends Logging {
   // for JVM off-heap overhead (thread stacks, native libraries, etc.)
   val DEF_HEAP_OVERHEAD_FRACTION = 0.1
   val MAX_JVM_GCTIME_FRACTION = 0.3
+  // Minimum amount of JVM heap memory to request per CPU core in megabytes
+  val MIN_HEAP_PER_CORE_MB: Long = 1024L
```

Comment:
Nit: Are those static constants common for all CSPs? If any of them is specific to a CSP, then we can define it as a function of the Platform.

Reply:
yes this is  for what we think is the minimum, common across them.